VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,54 @@
+
+AsyncAPI arises precisely from this basis. As an open government project, it advocates transparency and democratization in its system of governance and organization. This organizational system aims to empower people based on a horizontal decision-making structure.
+
+All relevant project data are public. The developed tasks are accessible and open to the participation of the community. This is intended to cultivate an environment of trust and respect for the user that encourages their involvement in pursuing a common goal. Collaboration and participation are key elements of a never-ending dynamic, an adaptable and flexible process where nothing is imposed or taken for granted.
+
+## Examples of tools that promote transparency in project work
+
+As mentioned above, the project is managed by all the people who actively collaborate in its community. AsyncAPI has several tools that ensure transparency in project management. Let's talk about them:
+
+- **Open governance**: To ensure that the project is managed by its community and that it’s the one pulling the strings, last year, it was hosted by an open government foundation, Linux Foundation. Hosting the project on a neutral foundation ensures true democratization for the project and its community. At the same time, it provides the long-term sustainability and success of the initiative. 
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
+
+- **Transparent communication and work**: To ensure that the principle of transparency governs communication, everything is said and done in public. The community uses Slack as the first communication platform, and secondly, GitHub is used as a means of communication and work. Therefore, all information about any project management is public, no matter how insignificant or essential it may be.
+
+- **Meetings**: There are different meeting points where the community can meet, propose or discuss any topic related to AsyncAPI. Any interested person can join these meetings and raise any thoughts. 
+
+  Currently, there is a bi-weekly general meeting, SIG Meeting, where the most significant changes of the project so far are presented and updated. Then there are three types of more specific arrangements: Contributor First, aimed at informing people who want to contribute, Spec 3.0 Meeting, where the progress and work of the version of the project are discussed. Spec 3.0 Meeting, where the project is concerned, and Spec 3.0 Meeting, where the project is discussed.
```

Comment:
don't mention meetings, this can change over time, 

more important focus on sentence that "even though we have synchronous meetings, we have a written rule that no decision making is done on the meetings. Presence on the meeting is not needed to actively participate. We value async communication more. After something is discussed on meeting, community needs to have time to react, watch the recording and decide."